### PR TITLE
[Feature] Private email field

### LINKED ISSUE #646: [Feature] Private email field
- [x] Onboarding
- [x] Employee/edit
- [x] Display in employee index
- [ ] Change "hire action" in ATS to use `emailPrivate`
- [x] Add `emailPrivate` to employee model
  - [x] Granite
  - [x] Igneous
  - [x] Quartz
    - [x] Add emailPrivate to "my information" tab

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #646

close with associatedemployers/quartz#83, associatedemployers/igneous#259

### CODE REVIEW

**Comment by JavaLavaMT** [comment]
employee/edit:
image

employee/index:
image

onboard: 
image

File: app/templates/account/employee/index/edit/index.hbs [hunk 1]
```diff
@@ -49,6 +49,10 @@
     <label for="email">Email</label>
     {{validated-input type="email" model=model valuePath="email" placeholder="Email" id="email"}}
   </div>
+  <div class="field">
+    <label for="privateEmail">Private Email</label>
+    {{validated-input type="email" model=model valuePath="privateEmail" placeholder="Private Email" id="privateEmail"}}
```
Comment (James1x0) [inline app/templates/account/employee/index/edit/index.hbs]:
```suggestion
    {{validated-input type="email" model=model valuePath="privateEmail" placeholder="Personal Email" id="privateEmail"}}
```

File: app/templates/account/employee/index/edit/index.hbs [hunk 2]
```diff
@@ -49,6 +49,10 @@
     <label for="email">Email</label>
     {{validated-input type="email" model=model valuePath="email" placeholder="Email" id="email"}}
   </div>
+  <div class="field">
+    <label for="privateEmail">Private Email</label>
```
Comment (James1x0) [inline app/templates/account/employee/index/edit/index.hbs]:
```suggestion
    <label for="privateEmail">Personal Email</label>
```

File: app/templates/account/employee/onboard/index.hbs [hunk 3]
```diff
@@ -1,14 +1,20 @@
...
+    {{#auto-focus}}
+      {{ui-calendar
+        class="field"
+        type="date"
+        startMode="year"
+        label="Birth Date"
+        dateValue=model.dateOfBirth
+        onChange=(action (mut model.dateOfBirth))}}
+    {{/auto-focus}}
+  </div>
```
Comment (James1x0) [inline app/templates/account/employee/onboard/index.hbs]: This is already in a field, does it need to be in another?

File: app/templates/account/employee/onboard/index.hbs [hunk 4]
```diff
@@ -1,14 +1,20 @@
...
+  <div class="field">
+    <label for="privateEmail">Private Email</label>
+    {{validated-input type="text" model=model valuePath='privateEmail' placeholder="Private Email" id="privateEmail"}}
```
Comment (James1x0) [inline app/templates/account/employee/onboard/index.hbs]:
```suggestion
    {{validated-input type="text" model=model valuePath='privateEmail' placeholder="Personal Email" id="privateEmail"}}
```
```suggestion
    {{validated-input type="text" model=model valuePath='privateEmail' placeholder="Private Email" id="privateEmail"}}
```

File: app/templates/account/employee/onboard/index.hbs [hunk 5]
```diff
@@ -1,14 +1,20 @@
...
+        class="field"
+        type="date"
+        startMode="year"
+        label="Birth Date"
+        dateValue=model.dateOfBirth
+        onChange=(action (mut model.dateOfBirth))}}
+    {{/auto-focus}}
+  </div>
+  <div class="field">
+    <label for="privateEmail">Private Email</label>
```
Comment (James1x0) [inline app/templates/account/employee/onboard/index.hbs]:
```suggestion
    <label for="privateEmail">Personal Email</label>
```

**Comment by JavaLavaMT** [comment]
My jaw dropped when i saw all these requests, glad to know they weren't anything too bad

File: app/templates/account/employee/onboard/index.hbs (hunk 3 above)
Comment (JavaLavaMT) [reply app/templates/account/employee/onboard/index.hbs]: yeah it does, its something weird, here are some screenshots

removing div on line #3 
image

removing the class in {{ui-calender}} but keeping div on line #3
image

with everything as the PR is 
image

Comment (JavaLavaMT) [reply app/templates/account/employee/onboard/index.hbs]: not sure if you can tell but on the 2nd img 'Birthdate' is not BOLD and it a little closer to the input bar

File: app/templates/account/employee/index/edit/index.hbs [hunk 6]
```diff
@@ -49,6 +49,10 @@
     <label for="email">Email</label>
     {{validated-input type="email" model=model valuePath="email" placeholder="Email" id="email"}}
   </div>
+  <div class="field">
+    <label for="privateEmail">Personal Email</label>
+    {{validated-input type="email" model=model valuePath="privateEmail" placeholder="Personal Email" id="privateEmail"}}
```
Comment (James1x0) [inline app/templates/account/employee/index/edit/index.hbs]:
```suggestion
    {{validated-input type="email" model=model valuePath="privateEmail" placeholder="Personal Email" id="private-email"}}
```

File: app/templates/account/employee/index/edit/index.hbs [hunk 7]
```diff
@@ -49,6 +49,10 @@
     <label for="email">Email</label>
     {{validated-input type="email" model=model valuePath="email" placeholder="Email" id="email"}}
   </div>
+  <div class="field">
+    <label for="privateEmail">Personal Email</label>
```
Comment (James1x0) [inline app/templates/account/employee/index/edit/index.hbs]:
```suggestion
    <label for="private-email">Personal Email</label>
```

File: app/templates/account/employee/onboard/index.hbs [hunk 8]
```diff
@@ -1,14 +1,20 @@
...
+        type="date"
+        startMode="year"
+        label="Birth Date"
+        dateValue=model.dateOfBirth
+        onChange=(action (mut model.dateOfBirth))}}
+    {{/auto-focus}}
+  </div>
+  <div class="field">
+    <label for="privateEmail">Personal Email</label>
+    {{validated-input type="text" model=model valuePath="privateEmail" placeholder="Personal Email" id="privateEmail"}}
```
Comment (James1x0) [inline app/templates/account/employee/onboard/index.hbs]:
```suggestion
    {{validated-input type="text" model=model valuePath="privateEmail" placeholder="Personal Email" id="private-email"}}
```

File: app/templates/account/employee/onboard/index.hbs [hunk 9]
```diff
@@ -1,14 +1,20 @@
...
+        class="field"
+        type="date"
+        startMode="year"
+        label="Birth Date"
+        dateValue=model.dateOfBirth
+        onChange=(action (mut model.dateOfBirth))}}
+    {{/auto-focus}}
+  </div>
+  <div class="field">
+    <label for="privateEmail">Personal Email</label>
```
Comment (James1x0) [inline app/templates/account/employee/onboard/index.hbs]:
```suggestion
    <label for="private-email">Personal Email</label>
```

**Pull request merged**